api/grpc_server.go

```diff
@@ -240,20 +242,48 @@ func NewGrpcService(port int, net NetworkAPI, state StateAPI, tx TxAPI, txMempoo
 			time.Minute * 3,
 		}),
 	}
+
+	nodeConfigParams := &pb.NodeConfigParams{
```

Comment:
I would extract this to a function just to reduce the length of this one and for clarity. something like
`NodeConfigParamsFromConfig(config.Config) *pb.NodeConfigParams`